tests/repositories/rest/test_incident.py

```
@@ -0,0 +1,99 @@
+            rsps.put(self.base_url)
+            repo.authenticated_put(self.base_url, body={})
+            self.assertEqual(rsps.calls[0].request.headers['Authorization'], f'Bearer {token}')
+
+    @parametrize(
+        ('status_code', 'risk', 'expected_result'),
+        [
+            (200, Risk.HIGH, True),  # Success case
+            (404, Risk.LOW, False),  # Not found
+        ],
+    )
+    def test_update_risk(self, status_code: int, risk: Risk, expected_result: bool) -> None:
```

Ruff

Boolean-typed positional argument in function definition

(FBT001)